report_thesis/src/sections/background.tex

```diff
@@ -330,46 +330,44 @@ \subsubsection{Gradient Boosting Regression (GBR)}\label{sec:gradientboost}
 While there are various choices for weak learners, decision trees are a common choice.
 Decision trees are a core component of gradient boosting methods. 
 They partition the data into subsets based on feature values, aiming to create groups where data points have similar predicted outcomes. 
-The quality of a split is measured using various metrics. Some common metrics include impurity, information gain, and Gini index. 
 
-Once optimal splits are identified, the tree is constructed by repeatedly partitioning the data until a stopping criterion is met.
+Once optimal splits based on some evaluation metric, are identified, the tree is constructed by repeatedly partitioning the data until a stopping criterion is met.
 The final model consists of splits that create distinct regions, each with a predicted response value based on the mean of the observations in that region.
 
-This leads us to \textit{Gradient Boosting}.
+This leads us to \textit{gradient boosting}.
 Instead of building a single decision tree, gradient boosting constructs multiple trees sequentially, with each new tree correcting the errors of the previous ones.
 Each tree is small, with few terminal nodes, preventing large adjustments based on a single tree's predictions. 
 It also ensures that each tree makes small and simple error corrections, such that each step refines the model's performance more reliably.
-Finally, small trees have the property of high bias, meaning that they are unable to capture complex relationships in the data and low variance.
-By combining multiple small trees, the model can capture complex relationships while maintaining a low variance.
 
-The model is constructed iteratively by adding trees to correct the residuals of the previous trees. 
-Initially, the prediction is set as $\hat{f}_0(x) = 0$ and residuals as $r_i = y_i$ for all $i$ in the training set, where $x$ represents the input feature(s) and $y$ is the true value or target variable. 
-At each iteration $b$, predictions are updated as:
+Initially, the prediction is set as $\hat{f}_0(\mathbf{x}) = 0$ and residuals as $r_i = y_i$ for all $i$ in the training set, where $\mathbf{x}$ represents the vector of input feature(s) and $y$ is the true value or target variable. 
+The model is then iteratively improved over $B$ iterations, where $B$ is the total number of trees specified during model configuration. 
+At each iteration $b \in \{1, 2, \ldots, B\}$, predictions are updated as:
```

Comment:
Each iteration $b$ from $1$ to $B$ updates the prediction ...

I think what you've written could be correct, but is this clearer?

Reply:
```suggestion
At each iteration $b$ from $1$ to $b$, predictions are updated as:
```